src/main/resources/application.properties

```diff
@@ -3,7 +3,7 @@ app.version=@project.version@
 app.displayName=HPO on FHIR
 spring.devtools.livereload.enabled=${SPRING_DEVTOOLS_LIVERELOAD_ENABLED:false}
 
```

Comment:
Probably not worth extensive refactoring, but we've been trying to move away from including environment variable references like this in properties files. They're usually redundant, and when they're not they cause inconsistency. Recent projects typically just specify the default:

```
server.servlet.context-path=/hpo-on-fhir
```